src/contracts/proxies/Vault721.sol

```diff
@@ -195,6 +253,19 @@ contract Vault721 is ERC721EnumerableUpgradeable {
     if (from != address(0)) {
       address payable proxy;
 
+      if (_allowlist[msg.sender]) {
+        if (
+          block.number < _hashState[firstTokenId].lastBlockNumber + blockDelay
```

Comment:
i believe that just calling it `tokenId` (since we're not using the batch transfer) could make it much more readable